src/locales/fr.json

```diff
@@ -70,7 +70,7 @@
   "contactToPlace": {
     "saveStartPlace": "Enregistrer le point de départ ?",
     "saveEndPlace": "Enregistrer le point d'arrivée ?",
-    "tooltip": "Pour automatiser cette catégorisation à l'avenir, enregistrer cette adresse dans une fiche contact",
+    "tooltip": "Pour fiabiliser l’automatisation de cette catégorisation à l’avenir, enregistrer cette adresse dans une friche contact.",
```

Comment:
s/friche/fiche
(now I understand the "wasteland"...)

Reply:
👍 🤦